imagesgallery/imagesgallery.py

```diff
@@ -65,6 +65,21 @@ def resource_string(self, path):
         data = pkg_resources.resource_string(__name__, path)
         return data.decode("utf8")
 
+    def read_file(self, path: str):
+        """Helper for reading a file using a relative path"""
+        file_content = ''
+        BASE_DIR = os.path.abspath(os.path.dirname(__file__))
+        file_path = os.path.join(BASE_DIR, path)
+
+        try:
+            with open(file_path, 'r') as file_data:
+                file_content = file_data.read()
+
+        except FileNotFoundError as e:
+            print(f"FileNotFoundError: {e}")
```

Comment:
I guess this error will not be retrieved to users, so you should update this to use the logger